### DISCUSSION TITLE
u04-mc-accuracy feedback

### DISCUSSION THREAD

**Discussion by agritter** [kcarnold/cs344-exam-23sp#26] [General]
@kangk9908
Hi Kun, 
I'm looking at your [question about improving accuracy](https://github.com/kcarnold/cs344-exam-23sp/blob/6a5024bc438f6db811ce74682ee7ba1fc4684112/u04-mc-accuracy/README.md?plain=1#L4), and I'm wondering why A is not correct? We've done some labs where adding more layers did improve the accuracy. In [`u06n3-nn-regression`](https://cs.calvin.edu/courses/cs/344/23sp/fundamentals/u06n3-nn-regression.ipynb), with one layer the model could only predict a straight line, despite the actual data being a curved line. But with a linear layer, a ReLU layer, and another linear layer the model was able to predict a line the followed the data fairly well.

Adding more layers may not always improve accuracy of the model, but I would argue that it sometimes can.

**Comment by chunkychoc0** [comment]
@agritter you are correct in saying that adding more layers can increase the accuracy... am I missing something?

**Reply by chunkychoc0** [reply]
Oh wait I see, I did not include it in the answers. I will fix the error! Thank you!
